shared/controller-api/src/main/proto/Controller.proto

```diff
@@ -113,7 +113,6 @@ message PingTxnStatus {
         OK = 0;
         LEASE_TOO_LARGE = 1;
         MAX_EXECUTION_TIME_EXCEEDED = 2;
-        SCALE_GRACE_TIME_EXCEEDED = 3;
```

Comment:
With protobufs, the typical course of action is to never reuse an identification number for a field. That's primarily to avoid compatibility issues. But, given that here are ignoring compatibility, should we renumber the fields to avoid gaps? If we are to follow the guidelines strictly, then we need to leave a placeholder so that we know we have used `3`.

Reply:
I think we should leave the placeholder with some comment in the proto file

Reply:
Ok.

Reply:
added deprecated flag in proto. it is much cleaner :)